test/kubemark/resources/heapster_template.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Yeah, it does - I just tried to move to RC, to see if it helps with validation errors (it didn't).

The are a couple of problems with reusing existing ones:
- we'd need to write a common denominator of config files (different sources/sinks)
- we'd need to figure out a way to make template for object. Currently we use jinja when creating an manifest on API server. For Kubemark I use sed, as it doesn't add any dependencies. I can move Kubemark to jinja, but it make running it harder (plus it'll take nontrivial amount of time, as I'll need to learn it).